scripts/common.py

```diff
@@ -625,6 +633,30 @@ def wait_for_port_to_be_open(port_number):
         sys.exit(1)
 
 
+def start_elasticsearch_server():
+    """Start the ElasticSearch server for running tests in development mode."""
+
+    # ElasticSearch server is only required in a development environment.
+    python_utils.PRINT('Starting ElasticSearch development server.')
+
+    # Clear previous data stored in the cluster.
+    if os.path.exists(ES_PATH_DATA):
+        os.remove(ES_PATH_DATA)
+
+    os.environ['ES_PATH_CONF'] = ES_PATH_CONF
+    subprocess.call(
+        ['%s/bin/elasticsearch' % ES_PATH, '-d', '-p'  '%s/pid' % ES_PATH])
```

Comment:
You have a couple spaces between the last two args, is that intentional?

Reply:
Fixed this